blogger.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strings"
+	"syscall"
+	"time"
+)
+
+var (
+	portAddr   = flag.String("http", ":8080", "HTTP listen address")
+	contentDir = flag.String("content", "content/", "Content directory")
+	index      = flag.Bool("index", false, "Show index")
+)
+
+func main() {
+	flag.Parse()
+
+	if strings.Index(*portAddr, ":") != 0 {
+		fmt.Printf("invalid port in address")
```

Comment:
Error messages should conventionally go to `os.Stderr` (use `fmt.Fprintf`).

Reply:
Noted.